app/assets/src/components/views/SampleViewV2/SampleViewV2.jsx

```diff
@@ -348,6 +362,33 @@ export default class SampleViewV2 extends React.Component {
     return parsedValue;
   };
 
+  adjustMetricPrecision = species => {
+    Object.entries(species).forEach(([countType, metricValue]) => {
+      if (countType in METRIC_DECIMAL_PLACES) {
+        species[countType] = parseFloat(metricValue).toFixed(
+          METRIC_DECIMAL_PLACES[countType]
+        );
+      } else if (countType === "nt") {
```

Comment:
This could almost be an else, but if you really want to do the sanity check do `if (countType === "nt" || countType === "nr")` and you can remove the bottom branch.

Reply:
Did you mean `if (key === "nt" || key === "nr")`?

Reply:
Yes, I did not go back to fully fix old comments :)